infinispan-source.kamelet.yaml

```diff
@@ -0,0 +1,80 @@
+spec:
+  sources:
+    - content: |-
+        public class InfinispanWorkaround extends org.apache.camel.builder.RouteBuilder {
+          @Override
+          public void configure() throws Exception {
+            getContext().setAutowiredEnabled(false);
+          }
+        }
+      name: InfinispanWorkaround.java
+  definition:
+    title: "Infinispan Source"
+    description: |-
+      Get Events from an Infinispan cache
+    required:
+      - hosts
+      - username
+      - password
+      - hostAddresses
+    properties:
+      hosts:
+        title: Hosts
+        description: Specifies the host of the cache on Infinispan instance
+        type: String
+        default: 1000
+      secure:
+        title: Secure
+        description: If the Infinispan instance is secured or not
+        type: boolean
+        default: true
+      username:
+        title: Username
+        description: Username to connect to Infinispan.
+        type: string
+      password:
+        title: Password
+        description: Password to connect to Infinispan.
+        type: string
+      saslMechanism:
+        title: SASL Mechanism
+        description: The SASL Mechanism to use
+        type: String
+        default: "DIGEST-MD5"
+      securityRealm:
+        title: Security Realm
+        description: Define the security realm to access the infinispan instance
+        type: string
+        default: "default"
+      securityServerName:
+        title: Security Server name
+        description: Define the security server name to access the infinispan instance
+        type: string
+        default: "infinispan"
+      eventTypes:
+        title: ElasticSearch cluster name
+        description: Specifies the set of event types to register by the consumer.Multiple event can be separated by comma. The possible event types are CLIENT_CACHE_ENTRY_CREATED, CLIENT_CACHE_ENTRY_MODIFIED, CLIENT_CACHE_ENTRY_REMOVED, CLIENT_CACHE_ENTRY_EXPIRED, CLIENT_CACHE_FAILOVER
+        type: string
+  flow:
+    from:
+      uri: "kamelet-reify:infinispan:{{cacheName}}"
+      parameters:
+        hosts: "{{hosts}}"
+        secure: "{{secure}}"
+        username: "{{username}}"
+        password: "{{password}}"
+        saslMechanism: "{{saslMechanism}}"
+        securityRealm: "{{securityRealm}}"
+        securityServerName: "{{securityServerName}}"
+        eventTypes: "{{eventTypes}}"
+      steps:
+        - to: kamelet:sink
```

Comment:
Don't know, is the result already serialized?

Reply:
The result is the exchange with some headers exposing the event. I can make it a json if needed.